app/models/game.rb

```diff
@@ -4,4 +4,38 @@ class Game < ApplicationRecord
   has_one :black_player, class_name: 'User', dependent: false
   has_one :winning_player, class_name: 'User', dependent: false
   has_many :pieces, dependent: false
+
+  attr_accessor :board_positions
+
+  def initialize
+    @board_positions = Array.new(8) { Array.new(8, ' ') }
+  end
+
+  def populate_board!
+    @board_positions[0][0] = Rook.new
+    @board_positions[0][1] = Knight.new
+    @board_positions[0][2] = Bishop.new
+    @board_positions[0][3] = Queen.new
+    @board_positions[0][4] = King.new
+    @board_positions[0][5] = Bishop.new
+    @board_positions[0][6] = Knight.new
+    @board_positions[0][7] = Rook.new
+
+    @board_positions[7][0] = Rook.new
+    @board_positions[7][1] = Knight.new
+    @board_positions[7][2] = Bishop.new
+    @board_positions[7][3] = Queen.new
+    @board_positions[7][4] = King.new
+    @board_positions[7][5] = Bishop.new
+    @board_positions[7][6] = Knight.new
+    @board_positions[7][7] = Rook.new
+
+    0.upto(7) do |b| # b to stand in for black for now
+      board[6][b] = Pawn.new
+    end
+
+    0.upto(7) do |w| # w to stand in for white for now
+      board[1][w] = Pawn.new
+    end
```

Comment:
For tests, you can setup expectations like this.

```Ruby
it 'adds correct number of Pawns to the board' do
  game = create :game
  game.populate_board!

  expect(game.pieces.where(type: 'Pawn', color: 'Black').count).to eq 8
  # just an example, may not be 100% correct code.
end
```

Reply:
@mscoutermarsh Thanks for the advice - Derek and I were wondering how to create a test(s) for this card.